x-pack/plugin/sql/qa/src/main/resources/subselect.sql-spec

```diff
@@ -0,0 +1,53 @@
+
```

Comment:
Please add the following comment in case someone needs to mute a test in the future:
`// To mute tests follow example in file: example.sql-spec`